src/main/kotlin/build/buf/gradle/BufPlugin.kt

```diff
@@ -35,6 +36,11 @@ class BufPlugin : Plugin<Project> {
     }
 
     private fun Project.configureBuf() {
+        repositories {
+            maven {
+                setUrl("https://s01.oss.sonatype.org/content/repositories/snapshots")
+            }
```

Comment:
There's a shorthand for this:
```suggestion
            maven(url = "https://s01.oss.sonatype.org/content/repositories/snapshots")
```

Reply:
I removed!